web/ui/templates/rules.html

```diff
@@ -34,6 +36,7 @@ <h2>Rules</h2>
               {{end}}
             </td>
             <td>{{humanizeDuration .GetEvaluationDuration.Seconds}}</td>
+            <td>{{.GetEvaluationTimestamp.UTC}}</h2></td>
```

Comment:
There is a reference for this cell:
```html
            <td class="last-scrape">
              {{if .LastScrape.IsZero}}Never{{else}}{{since .LastScrape}} ago{{end}}	              {{if .LastScrape.IsZero}}Never{{else}}{{since .LastScrape}} ago{{end}}
            </td>	            </td>
```

And `Timestamp` should prior to `Duration`.